mesop/components/table/table.ts

```diff
@@ -28,4 +32,18 @@ export class TableComponent {
   config(): TableType {
     return this._config;
   }
+
+  onClickCell(row: TableRow, index: number): void {
+    // On cell click events, return the column name and row Index so we can refer back
+    // to the same cell on the server.
+    const userEvent = new UserEvent();
+    userEvent.setHandlerId(this.config().getOnTableClickEventHandlerId()!);
+    userEvent.setKey(this.key);
+    const value = {
+      col_name: this._config.getDisplayedColumnsList()[index],
+      row_index: parseInt(row.getRowMap().get('Index')),
```

Comment:
This is a bit problematic because in downstream, I believe the Closure Compiler will minify these property names (e.g. "col_name" could become "a1"). You could technically wrap these in quotes (i.e. `"col_name:"`) to have the compiler not obfuscate them, but it's still not great because this isn't that typesafe (e.g. a typo in the object key here isn't caught at compile time).

A  different approach is to add a new field to UserEvent with the type `bytes` and then serialize/deserialize a table-specific proto message (defined in table.proto). This similar to what we do the [Type#bytes](https://github.com/google/mesop/blob/64467d43f00d807add09bcf9f77e5e460d039b65/mesop/protos/ui.proto#L203) field.

Reply:
Nice! Thanks for the great tip. That's definitely a lot better.